website/src/writing-php-code/phpdocs-basics.md

```diff
@@ -391,6 +391,8 @@ To make PHPStan understand the passed argument must be an instance of a `Resourc
 public function foo(Resource $var): void { ... }
 ```
 
+If you want to reference an internal type but have such a class in the same namespace, you get the error ``Class {Namespace}\Resource referenced with incorrect case: {Namespace}\resource.``. For resources, you can use ``closed-resource`` (or since 1.11.0 ``open-resource``) to avoid the name clash.
```

Comment:
`closed-resource` won't be the same type than `resource` though when analyzed with psalm which distinguish the resource states. `resource` would be `closed-resource|open-resource`.